Examples/TestText/main.js

```diff
@@ -19,6 +20,14 @@ wrappingLayer.x = Layer.root.x
 wrappingLayer.y = 300
 wrappingLayer.width = 100
 
+var alignmentLayer = new TextLayer()
+alignmentLayer.text = "align"
+alignmentLayer.fontSize = 64
+alignmentLayer.border = new Border({color: Color.red, width: 1})
+
 var h = new Heartbeat({handler: function(heartbeat) {
 	wrappingLayer.width = 100 + Math.sin(heartbeat.timestamp * 4) * 50
+	alignmentLayer.alignWithBaselineOf(wrappingLayer)
+	alignmentLayer.x = wrappingLayer.x + wrappingLayer.width*0.5 + alignmentLayer.width*0.5
```

Comment:
We also now have methods like `moveToRightOfSiblingLayer()` if that’s more useful than layout via centre/frame math :)